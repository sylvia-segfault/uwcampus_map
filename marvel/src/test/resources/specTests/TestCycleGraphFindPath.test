# Test the parser on a graph that contains three nodes with
# a cycle

LoadGraph graph TwoNodes.tsv
AddNode graph HAWK
AddEdge graph GORILLA-MAN HAWK AVF5
AddEdge graph HUMAN HAWK AVF5
FindPath graph GORILLA-MAN HAWK
FindPath graph HUMAN HAWK
