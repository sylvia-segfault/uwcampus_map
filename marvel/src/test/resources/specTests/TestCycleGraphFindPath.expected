# Test the parser on a graph that contains three nodes with
# a cycle

loaded graph graph
added node HAWK to graph
added edge AVF5 from GORILLA-MAN to HAWK in graph
added edge AVF5 from HUMAN to HAWK in graph
path from GORILLA-MAN to HAWK:
GORILLA-MAN to HAWK via AVF5
path from HUMAN to HAWK:
HUMAN to HAWK via AVF5